circuits.go

```diff
@@ -19,18 +19,17 @@ const (
 	AtomicQueryMTPCircuitID CircuitID = "credentialAtomicQueryMTP"
 	// AtomicQuerySigCircuitID is a type for credentialAttrQuerySig.circom
 	AtomicQuerySigCircuitID CircuitID = "credentialAtomicQuerySig"
-	// AtomicQueryMTPWithRelayCircuitID is a type for credentialAtomicQueryMTPWithRelay.circom
-	AtomicQueryMTPWithRelayCircuitID CircuitID = "credentialAtomicQueryMTPWithRelay"
-	// AtomicQuerySigWithRelayCircuitID is a type for credentialAttrQuerySigWithRelay.circom
-	AtomicQuerySigWithRelayCircuitID CircuitID = "credentialAtomicQuerySigWithRelay"
+	// AtomicQuerySigOnChainSmtCircuitID is a type for credentialAttrQuerySigOnChainSmt.circom
+	AtomicQuerySigOnChainSmtCircuitID CircuitID = "credentialAtomicQuerySigOnChainSmt"
 )
 
 // ErrorCircuitIDNotFound returns if CircuitID is not registered
 var ErrorCircuitIDNotFound = errors.New("circuit id not supported")
 
 const (
-	defaultMTLevels       = 32 // max MT levels, default value for identity circuits
-	defaultValueArraySize = 64 // max value array size, default value for identity circuits
+	defaultMTLevels        = 32 // max MT levels, default value for identity circuits
+	defaultMTLevelsOnChain = 32 // max MT levels on chain, default value for identity circuits
```

Comment:
In this context defaultMTLevels and defaultMTLevelsOnChain are the same

Reply:
resolved